doc/source/ray-core/scheduling/configure-memory-monitor.rst

```diff
@@ -0,0 +1,39 @@
+Configure memory monitor
+========================
+
+The memory monitor periodically checks the memory usage of the node and reports whether 
+the usage is above the threshold. The raylet uses this signal to evict worker processes to reduce the memory usage. This allows Ray to pre-emptively shed
+the load and avoid node failure. If the head node fails due to running out of memory it may take down the cluster.
+
+.. note::
+
+  The memory monitor is only available on Linux. It looks at the node-wide memory usage and limit and does not work with container memory limit. It still works when Ray runs in a container that does not have a memory limit. In this case its limit is equal to the node and the usage may include other containers.
```

Comment:
`does not work with container memory limit.` -> is it temporary? Maybe we should mention "currently"? Also we should probably mention this is an experimental feature (feel like we should support container case to call this feature > beta)